src/Biconomy.js

```diff
@@ -878,14 +881,14 @@ async function handleSendTransaction(engine, payload, end) {
               if (contractABI) {
                 let contract = new ethers.Contract(to, JSON.parse(contractABI), engine.ethersProvider);
                 let methodSignature = methodName+"("+typeString+")";
-                gasLimit = await contract.estimateGas[methodSignature](...paramArrayForGasCalculation, { from: account });
+                txGas = await contract.estimateGas[methodSignature](...paramArrayForGasCalculation, { from: account });
                  // do not send this value in API call. only meant for txGas
-                gasLimitNum = ethers.BigNumber.from(gasLimit.toString())
+                gasLimitNum = ethers.BigNumber.from(txGas.toString())
                  .add(ethers.BigNumber.from(5000))
                  .toNumber();
 
-                _logMessage(`Gas limit calculated for method ${methodName} in SDK: ${gasLimitNum}`);
-                _logMessage("gas limit number" + gasLimitNum);
+                _logMessage(`Gas limit (txGas) calculated for method ${methodName} in SDK: ${gasLimitNum}`);
```

Comment:
gasLimitNum is logged twice with different wordings here. It should be logged once.

Reply:
done